.github/workflows/2_bump_versions.yml

```diff
@@ -0,0 +1,55 @@
+name: Bump versions
+
+on:
+  workflow_dispatch:
+    inputs:
+      npm_channel:
+        description: 'Prisma CLI NPM Channel'
+        required: true
+      version:
+        description: 'New Prisma CLI version'
+        required: true
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  ENVIRONMENT: ${{ secrets.ENVIRONMENT }}
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v2
+      - name: Install Semver # Needed to use semver in scripts
+        run: npm install
+      - name: Setup Git
+        run: |
+          git config --global user.email "prismabots@gmail.com"
+          git config --global user.name "Prismo"
+      - name: Update versions in scripts folder 
+        run: npm run bump_prisma_version_in_scripts ${{ github.event.inputs.npm_channel }} ${{ github.event.inputs.version }}
+      - name: Update next extension version in scripts folder
+        id: update
```

Comment:
What is this `id` used for? Maybe leave information as comment behind so does not get accidentally removed.

Reply:
It's used to access output values from this step in later workflow steps. Added a comment as well.